lib/core/layout/components/header/header.component.ts

```diff
@@ -49,18 +49,29 @@ export class HeaderLayoutComponent implements OnInit {
     @Input() showSidenavToggle: boolean = true;
 
     /** Emitted when the sidenav button is clicked. */
-    @Output() clicked = new EventEmitter<any>();
+    @Output() clicked = new EventEmitter<boolean>();
+
+    @Input() initialSidenavExpanded: boolean = true;
+
+    sidenavExpanded: boolean = true;
```

Comment:
Add the @Input attribute to allow binding from the Layout
```suggestion
    @Input() sidenavExpanded: boolean = true;
```